Cargo.toml

```diff
@@ -4,12 +4,22 @@ members = [
     "bindings/wysiwyg-wasm",
     "crates/wysiwyg",
     "crates/matrix_mentions",
+    "uniffi-bindgen",
+]
+default-members = [
+    "crates/wysiwyg",
+    "crates/matrix_mentions",
 ]
 resolver = "2"
 
 [workspace.package]
 rust-version = "1.71"
 
+[workspace.dependencies]
+uniffi = { git = "https://github.com/aringenbach/uniffi-rs.git", rev = "7fea5def9b7a00cec35b94bd1916172cd0ca60fb" }
```

Comment:
Could you add a comment linking to the issue in UniFFI to help explain why this is needed?

Reply:
Added to description